seatunnel-connectors-v2/connector-kudu/src/main/java/org/apache/seatunnel/connectors/seatunnel/kudu/kuduclient/KuduInputFormat.java

```diff
@@ -116,7 +118,7 @@ public static SeaTunnelRow getSeaTunnelRowData(RowResult rs, SeaTunnelRowType ty
             } else if (BasicType.STRING_TYPE.equals(seaTunnelDataType)) {
                 seatunnelField = rs.getString(i);
             } else {
-                throw new IllegalStateException("Unexpected value: " + seaTunnelDataType);
+                throw new KuduConnectorException(KuduConnectorErrorCode.GET_KUDU_VALUE_FAILED, "Unexpected value: " + seaTunnelDataType);
```

Comment:
CommonErrorCode.UNSUPPORTED_DATA_TYPE is better, remove this unclear error code.